src/ebusd/mqtthandler.cpp

```diff
@@ -50,9 +51,10 @@ static const struct argp_option g_mqtt_argp_options[] = {
   {"mqttpass",    O_PASS, "PASSWORD",    0, "Use PASSWORD when connecting to MQTT broker (no default)", 0 },
   {"mqtttopic",   O_TOPI, "TOPIC",       0,
    "Use MQTT TOPIC (prefix before /%circuit/%name or complete format) [ebusd]", 0 },
-  {"mqttretain",  O_RETA, nullptr,       0, "Retain all topics instead of only selected global ones", 0 },
-  {"mqttjson",    O_JSON, nullptr,       0, "Publish in JSON format instead of strings", 0 },
-  {"mqttignoreinvalid", O_IGIN, nullptr, 0,
+  {"mqttretain",   O_RETA, nullptr,       0, "Retain all topics instead of only selected global ones", 0 },
+  {"mqttjson",     O_JSON, nullptr,       0, "Publish in JSON format instead of strings", 0 },
+  {"mqttv311",     O_V311, nullptr,       0, "Use MQTT protocol 3.1.1 rather than 3.1", 0 },
```

Comment:
rather mqttversion with "VERSION" as int
"Use MQTT protocol VERSION instead of library default"